tests/default.nf.test

```diff
@@ -27,8 +28,20 @@ nextflow_pipeline {
                     // All stable path name, with a relative path
                     stable_name,
                     // All files with stable contents
-                    stable_path
-                ).match() }
+                    stable_path,
+                    // Important files with unstable content
+                    path("${params.outdir}/samtools/megahit.SAMPLE1_PE.flagstat").readLines().size(),
```

Comment:
afaik `$outputDir` should be rather used
```suggestion
                    path("$outputDir/samtools/megahit.SAMPLE1_PE.flagstat").readLines().size(),
```
also true for all below

Reply:
Thanks! I'll fix them.